### PR TITLE
docs: update web.dev links to new developer.chrome.com home

### PR DESCRIPTION
**Pull request by brendankenny**
Update remainder of web.dev docs links to new https://developer.chrome.com/docs/lighthouse/ home

**Related Issues/PRs**
#14400

### CODE REVIEW

File: docs/v8-perf-faq.md [hunk 1]
```diff
@@ -132,8 +132,7 @@ experience is best practice. [Read more about field & lab](https://developers.go
 
 ### How should I work to optimize CLS differently given that it has been updated?
 
-The [windowing
-adjustment](https://www.google.com/url?q=https://web.dev/evolving-cls/&sa=D&source=editors&ust=1622570731600000&usg=AOvVaw2R7Y5uFrQX7Mpdj__5SdYq)
```
Comment (brendankenny) [inline docs/v8-perf-faq.md:136]: not sure how we missed these :)

File: core/audits/preload-lcp-image.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const UIStrings = {
   title: 'Preload Largest Contentful Paint image',
   /** Description of a lighthouse audit that tells a user to preload an image in order to improve their LCP time.  */
   description: 'If the LCP element is dynamically added to the page, you should preload the ' +
-    'image in order to improve LCP. [Learn more about preloading LCP elements](https://web.dev/optimize-lcp/#preload-important-resources).',
+    'image in order to improve LCP. [Learn more about preloading LCP elements](https://web.dev/optimize-lcp/#optimize-when-the-resource-is-discovered).',
```
Comment (adamraine) [inline core/audits/preload-lcp-image.js:20]: Are we gong to create our own doc for this as part of https://github.com/GoogleChrome/lighthouse/issues/13738?

File: core/audits/seo/is-crawlable.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const UIStrings = {
   failureTitle: 'Page is blocked from indexing',
   /** Description of a Lighthouse audit that tells the user *why* allowing search-engine crawling of their page is beneficial. This is displayed after a user expands the section to see more. No character length limits. The last sentence starting with 'Learn' becomes link text to additional documentation. */
   description: 'Search engines are unable to include your pages in search results ' +
-      'if they don\'t have permission to crawl them. [Learn more about crawler directives](https://web.dev/is-crawable/).',
```
Comment (adamraine) [inline core/audits/seo/is-crawlable.js:27]: oof on the mispell

**Review by adamraine — APPROVED** [review]

**Review by connorjclark — APPROVED** [review]

File: core/audits/preload-lcp-image.js (hunk 2 above)
Comment (brendankenny) [reply core/audits/preload-lcp-image.js:20]: yes

File: core/audits/seo/is-crawlable.js (hunk 3 above)
Comment (brendankenny) [reply core/audits/seo/is-crawlable.js:27]: Ha, well spotted. I was confused at first how it worked at all (turns out https://github.com/GoogleChrome/web.dev/pull/5407 added a redirect) and then worried for a second that we had our audit ID wrong all this time.

**Pull request merged**